src/components/Diet/Layout.js

```diff
@@ -1,33 +1,15 @@
-import React, { useEffect, useState } from 'react';
-import { useSelector, useDispatch } from 'react-redux';
-import PropTypes from 'prop-types';
-import { getDietAvailable } from '../../selectors/diet/getDietAvailable';
-import { fetchDiet } from '../../actions/diet/fetchDiet';
-import { setRouter } from '../../actions/router/setRouter';
-import { fetchRouter } from '../../actions/router/fetchRouter';
-
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { getAvailableDiet } from '../../selectors/firebase/getAvailableDiet';
 import { Diet } from './Diet';
 import { Basket } from './Basket';
 import { BlankSlate } from '../Common/BlankSlate';
 import { applyKeyboardNavigation } from '../../utils/applyKeyboardNavigation';
 import { B_KEY_CODE, D_KEY_CODE } from '../../consts/keyboard-key-codes';
 
-export const Layout = ({ location }) => {
-  const dietAvailables = useSelector(getDietAvailable);
+export const Layout = () => {
+  const availableDiet = useSelector(getAvailableDiet);
   const [selectedTab, setSelectedTab] = useState('diet');
-  const dispatch = useDispatch();
```

Comment:
this was the logic for getting the diet loaded by URL but as it's not working right now we can remove it and redo it again in a better way.
I think we need to replace the default `route` configuration by the user profile or a welcome page